src/platform/testing/unit/mocha-setup.js

```diff
@@ -8,12 +8,11 @@ import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { JSDOM } from 'jsdom';
 import '../../site-wide/moment-setup';
+import ENVIRONMENTS from '../../../site/constants/environments';
+
 // import sinon from 'sinon'
 
-global.__BUILDTYPE__ = process.env.BUILDTYPE || 'vagovdev';
-global.__ALL_CLAIMS_ENABLED__ =
-  global.__BUILDTYPE__ === 'vagovdev' ||
-  process.env.ALL_CLAIMS_ENABLED === 'true';
```

Comment:
I searched, and I didn't see any instances of code checking for `__ALL_CLAIMS_ENABLED__`.